ui-tests/src/test/resources/warnings_ng_plugin/source-view/SampleClassWithUnconventionalPackageNaming.java

```diff
@@ -0,0 +1,5 @@
+package COM.hm.hafner.analysis.complex.naming.structure;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Package name is not same as directory.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)